### ISSUE TITLE
MLP output

### ISSUE THREAD

**Issue by maarezaei** [YassineYousfi/alaska#5]
In this file: "tf_extract_features_color_separated.ipynb" which is for test, the variable stego_schemes includes only steganography methods, not "COVER":

stego_schemes = ['EBS', 'JUNI', 'NSF5', 'UED']

The variable "logits" as the output of test:

logits = sess.run(y, feed_dict={x: test_feature_map})

includes 5 numbers.

How  "stego_schemes[np.argmax(softmax(logits))]" can give "COVER" as the output while we don't have it in the list in "stego_schemes "?

Now when I use Models (QF=95) and the image file "iPAD-pro-7.1-13inch_2065.jpg", I always get "JUNI" as result. It doesn't matter the input is COVER, EBS, JUNI, NSF5 or UED.

**Comment by YassineYousfi** [comment]
We are aware of this issue, and working to fix it
